prepare-vms/scripts/postprep.rc

```diff
@@ -31,22 +31,29 @@ STEP = 0
 START = time.time()
 
 def bold(msg):
-    return "{} {} {}".format("$(tput smso)", msg, "$(tput rmso)")
+    return "{} {} {}".format("", msg, "")
 
-def system(cmd):
+def system(cmd, ok_codes=[]):
```

Comment:
What about using `[0]` as the default value for `ok_codes`?

Reply:
In that case, I assume you would also want to change this:
`      if ret["retcode"] != 0 and ret["retcode"] not in ok_codes:`
to:
`      if ret["retcode"] not in ok_codes:`

In that case, it would require changing the usage from:
* `ok_codes=[2304]`

to one of:
* `ok_codes=[0, 2304]`
* `ok_codes.append(2304)`

...which seems less friendly from a UX perspective, no?